projects/mercury/src/file/DirectoryInformationDrawer.js

```diff
@@ -235,9 +236,21 @@ const PathMetadata = React.forwardRef((
     ref
 ) => {
     const {data, error, loading} = useAsync(() => LocalFileAPI.stat(path, showDeleted), [path]);
-    const {hierarchy} = useContext(VocabularyContext);
+    const {hierarchy, vocabulary} = useContext(VocabularyContext);
     const [updateDate, setUpdateDate] = useState(Date.now());
 
+    const useStyles = makeStyles((theme) => ({
+        typeNameStyle: {
+            float: 'right',
+            fontSize: '0.8em',
+            marginTop: 2,
+            borderColor: 'none',
+            borderWidth: 0,
+            borderStyle: 'none',
+            opacity: 0.4
+        },
+    }));
+    const classes = useStyles();
```

Comment:
we usually import the styles as part of the class export (last line):
`export default withRouter(withStyles(styles)(ContextualDirectoryInformationDrawer));
`
then the `classes` property is part of the props of DirectoryInformationDrawer.